spec/latest/core/index.html

```diff
@@ -1250,8 +1250,47 @@ <h3>Description of HTTP Status Codes and Errors</h3>
       be given an identifier. When dereferenced, the <i>Error</i> resource can provide
       more detailed information or possible ways to resolve the problem, if applicable.<p/>
 
+    <p>Finally, the server SHOULD provide error descriptions using an [[!RFC7807]] standard
+      by using an <i>application/problem+json</i> response. When doing so, the server also MUST provide
+      an additional header pointing to either the built-in Hydra <i>http://www.w3.org/ns/hydra/error</i>
+      context or any JSON-LD context that maps the terms <i>type</i>, <i>title</i>, <i>detail</i>,
+      <i>status</i> and <i>instance</i> the same way as the standard one.</p>
+
+    <pre class="example nohighlight" data-transform="updateExample"
+         title="RFC-7807 compatible error description">
+      <!--
+      HTTP/1.1 400 Bad Request
+      Content-Type: application/problem+json
+      Link: <http://www.w3.org/ns/hydra/error>; rel="http://www.w3.org/ns/json-ld#context"
+
+      {
+        "type": "https://example.com/probs/out-of-credit",
```

Comment:
Given that `type` is mapped to `@type` in the error context this will cause an error

Reply:
Works just fine in the playground. Also in the legacy 1.0 version

Reply:
Actually - in 1.0 it seems to throw, see https://json-ld.org/playground/1.0/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%7B%22rdfs%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23%22%2C%22hydra%22%3A%22http%3A%2F%2Fwww.w3.org%2Fns%2Fhydra%2Fcore%23%22%2C%22type%22%3A%22%40type%22%2C%22title%22%3A%22rdfs%3Alabel%22%2C%22detail%22%3A%22rdfs%3Acomment%22%2C%22status%22%3A%22hydra%3AstatusCode%22%2C%22instance%22%3A%7B%22%40id%22%3A%22rdfs%3AseeAlso%22%2C%22%40type%22%3A%22%40id%22%7D%7D%2C%22type%22%3A%22https%3A%2F%2Fexample.com%2Fprobs%2Fout-of-credit%22%2C%22%40type%22%3A%22http%3A%2F%2Fwww.w3.org%2Fns%2Fhydra%2Fcore%23Error%22%2C%22title%22%3A%22You%20do%20not%20have%20enough%20credit.%22%2C%22detail%22%3A%22Your%20current%20balance%20is%2030%2C%20but%20that%20costs%2050.%22%2C%22instance%22%3A%22%2Faccount%2F12345%2Fmsgs%2Fabc%22%2C%22balance%22%3A30%2C%22accounts%22%3A%5B%22%2Faccount%2F12345%22%2C%22%2Faccount%2F67890%22%5D%7D. Thank's @lanthaler - I'll need to make some more experiments with it

Reply:
That's weird. I tried this earlier and I thought it was working 😔. 

We can recommend JSON-LD 1.1 but probably a bad idea to have it as a requirement?

Alternatively `"type": ["https://example.com/probs/out-of-credit", "hydra:Error"]` is fine in JSON-LD terms but I thank is invalid problem+json

Reply:
Good question - can a problem+json point to multiple types using array ?

Reply:
Nope, `type` is a `string` that should be in the form of a URI. Is it impossible to map this in JSON-LD somehow? Or would that require Hydra to demand JSON-LD version 1.1?

Reply:
What is the state of JSON-LD 1.0 specification? When I browsed that spec all I can see a popup with "This version is outdated!". Does it mean 1.0 is deprecated and 1.1 should be used? Like in case of RFC 7230 that makes the older one, RFC 2616 obsolete?

Reply:
I would suggest to not require Hydra clients to understand JSON-LD 1.1 just for this. The pragmatic solution would be to map RFC7807's `type` to something else than `@type` aka `rdf:type`. Perhaps `hydra:errorType`... or introduce a separate vocab for RFC 7807 altogether

Reply:
Obviously! Thank you @lanthaler

Reply:
I don't see how having a separate vocab solves the type mapping issue - there would also be some term to be mapped and the context to be usage agnostic.
Also mapping it to `rdf:type` feels like a hack.
Enforcing JSON-LD 1.1 is neither an option for me.
I'd go with a statement that response of type `application/problem+json` using this context should be considered `hydra:Error` and just make a note for the example that the `"@type": "http://www.w3.org/ns/hydra/core#Error"` is optional and JSON-LD 1.1 compatible. Or maybe we can get rid of it completely for sake of simplicity.

Reply:
> for sake of simplicity.

`rdf:type` **is** the simple solution. It just works and requires no sacrifice or weird conditionals in the spec. Why do you say it's hack? `@type` is just a syntactic sugar equivalent to using `rdf:type` directly.

Reply:
`rdf:type` seems to be just the type of glue we need here. 👍🏼 from me.

Reply:
OK, `rdf:type` it is - I'll update the PR ASAP